Client/Frontend/Browser/Tab.swift

```diff
@@ -211,7 +220,8 @@ class Tab: NSObject {
 
             self.webView = webView
             self.webView?.addObserver(self, forKeyPath: KVOConstants.URL.rawValue, options: .new, context: nil)
-            self.userScriptManager = UserScriptManager(tab: self, isFingerprintingProtectionEnabled: Preferences.Shields.fingerprintingProtection.value, isCookieBlockingEnabled: Preferences.Privacy.blockAllCookies.value, isU2FEnabled: webView.hasOnlySecureContent)
+            self.userScriptManager = UserScriptManager(tab: self, isFingerprintingProtectionEnabled: Preferences.Shields.fingerprintingProtection.value, isCookieBlockingEnabled: Preferences.Privacy.blockAllCookies.value, isU2FEnabled: webView.hasOnlySecureContent,
```

Comment:
Break this into multiple lines at this point:

```suggestion
            self.userScriptManager = UserScriptManager(
                tab: self, 
                isFingerprintingProtectionEnabled: Preferences.Shields.fingerprintingProtection.value, 
                isCookieBlockingEnabled: Preferences.Privacy.blockAllCookies.value, 
                isU2FEnabled: webView.hasOnlySecureContent,
```